docs/docs/workflows/recover-coordinator.md

```diff
@@ -3,6 +3,12 @@
 Different situations require the [recovery](../features/recovery.md) of the Coordinator.
 If the Coordinator fails to unseal its state, it will enter recovery mode.
 
+:::tip
+
+Use [distributed Coordinator instances](../features/recovery.md#distributed-coordinator) to minimize situations where manual recovery is required.
```

Comment:
I'm unsure if "minimize situations" is the proper term here.
Some suggestions:
* reduce the frequency
* diminish the likelihood
* lessen the occurrences